sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_multivariate_detect.py

```diff
@@ -62,21 +63,35 @@ def train(self, start_time, end_time, max_tryout=500):
 
         # Wait until the model is ready. It usually takes several minutes
         model_status = None
-        tryout_count = 0
-        while (tryout_count < max_tryout and model_status != "READY"):
-            model_status = self.ad_client.get_multivariate_model(trained_model_id).model_info.status
-            tryout_count += 1
-            time.sleep(2)
 
-        assert model_status == "READY"
+        start = time.time()
+        while time.time() - start < timeout and model_status != ModelStatus.READY and model_status != ModelStatus.FAILED:
+            model_info = self.ad_client.get_multivariate_model(trained_model_id).model_info
+            model_status = model_info.status
+            time.sleep(1)
 
-        print("Done.", "\n--------------------")
-        print("{:d} available models after training.".format(len(new_model_list)))
+        if model_status == ModelStatus.FAILED:
+            print("Creating model failed.")
+            print("Errors:")
+            if model_info.errors:
+                for res in model_info.errors:
+                    print("Error code: {}. Message: {}".format(res.code, res.message))
+            else:
+                print("None")
+            return None
 
-        # Return the latest model id
-        return trained_model_id
+        if model_status == ModelStatus.READY:
+            print("Done.", "\n--------------------")
```

Comment:
`print("Done.\n--------------------")`

Reply:
movev the `new_model_list = list(self.ad_client.list_multivariate_model(skip=0, top=10000))` here